metrics.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: base
+permalink: /metrics/
+---
+
+## Metrics -- DRAFT
+
+*Metric: a standard of measurement* -- [Merriam-Webster](https://www.merriam-webster.com/dictionary/metric)
+
+This page will contain summaries of the metrics that the Arbiter uses to assess forecasts. Check back soon for more information.
+
+Please send feedback to Will Holmgren at <a href="mailto:holmgren@email.arizona.edu">holmgren@email.arizona.edu</a>
+
+
+Deterministic
+-------------
+
+* MAE: Mean absolute error
+* MAPE: Mean absolute percentage error
+* MBE: Mean bias error
+* RMSE: Root mean squared error
+* NRMSE: Normalized root mean squared error
+* CRMSE: Centered (unbiased) root mean squared error
```

Comment:
Should add variance/std. dev. here if we're including this (presumably for the Taylor diagrams)